src/lib/krb5/os/accessor.c

```diff
@@ -41,7 +41,7 @@ krb5_error_code KRB5_CALLCONV
 krb5int_accessor(krb5int_access *internals, krb5_int32 version)
 {
     if (version == KRB5INT_ACCESS_VERSION) {
-#if DESIGNATED_INITIALIZERS
+#ifdef DESIGNATED_INITIALIZERS
```

Comment:
This change would break the intent of the code, which is to use designated initializers only for C99 compilers or suitable versions of gcc.  DESIGNATED_INITIALIZERS is always defined by isn't always true.  What is the warning related to this change?